src/Compiler/Service/ServiceParsedInputOps.fs

```diff
@@ -1479,7 +1479,7 @@ module ParsedInput =
                         | SyntaxNode.SynExpr(SynExpr.Record(None, _, fields, _)) :: _ ->
                             let isFirstField =
                                 match field, fields with
-                                | Some contextLid, SynExprRecordField(fieldName = lid, _) :: _ -> contextLid.Range = lid.Range
```

Comment:
The existing pattern matching code seems incorrect - it was missing the tuple parentheses that should have been there. `fieldName = lid, _`  should have been `fieldName = (lid, _)`.